### DISCUSSION TITLE
Github Streak not updating.

### DISCUSSION THREAD

**Discussion by JunaidKhan444** [community/community#77936] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My contribution on December 2nd is not registered, leading to a disruption in my GitHub streak.I have thoroughly reviewed my commits and repository activity, but I am unable to pinpoint the reason for this oversight.

**Comment by ColinHuelsbusch** [comment]
Hi,

you can use `git commit --date=<date>` .

Kind regards,
Colin

**Reply by JunaidKhan444** [reply]
Thanks @ColinHuelsbusch for the reply. I made that commit on December 2; that branch was merged and deleted.

Now I don't know how to use this command..

**Reply by ColinHuelsbusch** [reply]
You can make another change and commit it with this command:
`git commit --date="2023-12-02T12:00:00" -m "Your commit message"` 
Push that to the repository and it will create a commit on December 2nd.

**Reply by JunaidKhan444** [reply]
Thanks but it is not working.
